ambry-api/src/main/java/com.github.ambry/network/ClientNetworkRequestMetrics.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.network;
+
+import com.github.ambry.metrics.MetricsHistogram;
+
+
+/**
+ * Tracks a set of metrics for a network request by a client
+ */
+public class ClientNetworkRequestMetrics implements NetworkRequestSend {
+  private MetricsHistogram requestSendTime;
```

Comment:
we usually use `Histogram` from codahale directly. Why use this? Its implementation is eventually using a `Histogram` anyway.